tests/astrapy/test_ops.py

```diff
@@ -83,7 +83,7 @@ def test_create_database(devops_client):
 @pytest.mark.describe("should create a keyspace")
 def test_create_keyspace(devops_client):
     response = devops_client.create_keyspace(
-        keyspace="test_namespace", database=os.environ["ASTRA_DB_ID"]
+        keyspace="test_namespace", database=str(uuid.uuid4())
```

Comment:
I guess such a test will never work. The DB_ID must exist on the test account - a random string will not. Unless I'm missing something, this should be reverted (to trusting the test to run with the DASTRA_DB_ID variable set to an existing database ID in the tester's account)

Reply:
Oh this was a mistake on my part! See, i was trying to strip out the use of ASTRA_DB_ID where it was still used - but, you're right, we actually need a test db to exist. I think i just want to clarify that this is not necessarily the ASTRA_DB_ENDPOINT_URL database.... so maybe it can be called ASTRA_DB_TEST_ID or something...